GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -109,6 +109,8 @@ gdjs.LayerPixiRenderer.prototype.addEffect = function (effectData) {
     update: filterCreator.update,
   };
 
+  // @ts-ignore PIXI isn't typed for now.
+  if (this._isLightingLayer) filter.pixiFilter.blendMode = PIXI.BLEND_MODES.ADD;
```

Comment:
@4ian I guess if the layer is a lighting layer we should be having the ADD blend mode for filters (always?) 🤔.

Reply:
ADD or MULTIPLY? But yeah I guess all filters should then use it.. or just the container?

Reply:
Filters are affecting the light objects, so they should he having ADD blend mode. (They are applied on the layer's container, however the container is meant to contain only light objects.) So, if the filters would have any other blend mode, they will basically be overriding light object's blend mode.

Reply:
> or just the container?

Only the filters applied on the lighting layer's container should be having ADD blend mode, we shouldn't really change anything else I guess.

Reply:
Ok make sense! I was wondering if there was possibility of using ADD blend mode on the container only (rather than the filters + individual light objects) (and still have a multiply on the render texture).
Sounds fine to use ADD on filters/objects if it's not possible :)

Reply:
> I was wondering if there was possibility of using ADD blend mode on the container only (rather than the filters + individual light objects) (and still have a multiply on the render texture).

I didn't think about this earlier 🤔 I'll inform you after trying this out :)

Reply:
Let me know if you can try this :)